TcgPocket/App/src/components/Navigation/PrimaryNavigation.tsx

```diff
@@ -82,21 +80,25 @@ export function PrimaryNavigation({
           </Menu>
         </Flex>
       </Navbar>
-      {children}
-    </div>
+      <Box sx={childContainerSx}>{children}</Box>
+      <Center sx={footerSx}>(≖ᴗ≖✿)</Center>
+    </Box>
   );
 }
 
 function navbarSx(theme: MantineTheme): CSSObject {
   return {
-    borderBottom: `2px solid ${theme.colors.blue[3]}`,
+    borderBottom: `1px solid ${theme.colors.blue[3]}`,
+    boxShadow: `0px 0px 4px ${theme.colors.blue[5]}`,
     display: 'flex',
     flexDirection: 'row',
     justifyContent: 'space-between',
     alignItems: 'center',
     padding: '0rem 2rem 0rem 1rem',
     overflow: 'hidden',
```

Comment:
![image](https://github.com/gbrad777/TcgPocket/assets/113717324/3d0a351d-4eae-49f9-8ba4-918b54672b42)

With the changes made, the user icon dropdown is cut off; I found that if you change the overflow for the navbar styling to 'inherit', it fixes this. 😄

Reply:
![image](https://github.com/gbrad777/TcgPocket/assets/113717324/64e8d53e-8a3a-4d2f-896b-38e9621ba069)

*With the changes made, the user icon dropdown is cut off.

```suggestion
    overflow: 'inherit',
```

😄